packages/docs/docs/communications/organizing-communications.md

```diff
@@ -0,0 +1,480 @@
+# Organizing Communications
+
+# Introduction
+
+Communications in a healthcare context (modeled as [Communication](/docs/api/fhir/resources/communication) resources) can include many scenarios – patient to physician, physician to physician, device to physician, and more. This results in a large volume of communications within this context, so ensuring that they are well organized is vital. For example, a provider may have staff on call to respond to patient messages. Without proper organization, messages may not reach the correct staff member.
```

Comment:
Our style guide is that all references to a resource name by itself (.e.g "Communication" or Communications), should be in backticks (e.g. [`Communication`](/docs/api/fhir/resources/communication))